src/omotes_simulator_core/solver/utils/fluidprop.py

```diff
@@ -0,0 +1,523 @@
+# code copied from: https://github.com/Dennis-van-Gils/python-fluidprop/blob/main/src/fluidprop.py
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Easy access to thermodynamic fluid properties as a function of temperature
```

Comment:
I a direct copy the best thing to do?

Reply:
I agree, this will make it easier to update to newer versions, and this makes it very clear that this is code not created by us. I also excluded it from code quality checks

Reply:
Remove comman line interface and pprint/report method.